README.md

```diff
@@ -1,17 +1,19 @@
 # MerkleMap
 
+[![Run in Livebook](https://livebook.dev/badge/v1/blue.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2Fderekkraan%2Fmerkle_map%2Fblob%2Fmaster%2Fmerkle_map.livemd)
```

Comment:
This won't work until merged, but a working example:

[![Run in Livebook](https://livebook.dev/badge/v1/blue.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2FNezteb%2Fmerkle_map%2Fblob%2Fdocs%2Fexamples%2Fmerkle_map.livemd)